Source/GSXib5KeyedUnarchiver.m

```diff
@@ -3186,6 +3186,20 @@ - (id) decodeObjectForXib: (GSXibElement*)element
       else if ([object respondsToSelector: @selector(setHeaderToolTip:)])
         [object setHeaderToolTip: [element attributeForKey: @"toolTip"]];
     }
+  else if ([element elementForKey: @"toolTip"])
+    {
+      GSXibElement *toolTipElement = [element elementForKey: @"toolTip"];
+      NSString *v = [toolTipElement value];
+      if ([object respondsToSelector: @selector(setToolTip:)])
```

Comment:
We should try to share some code with the lines above by extracting the whole `if` block into a new method.

Reply:
Good point. I will do that asap.

Reply:
I have done this, per your suggestion.  Resolve this comment, if you feel as though it fits what you wanted.